packages/react/src/__tests__/ProgressBar.test.tsx

```diff
@@ -6,7 +6,20 @@ import axe from 'axe-core'
 import {FeatureFlags} from '../FeatureFlags'
 
 describe('ProgressBar', () => {
-  behavesAsComponent({Component: ProgressBar, toRender: () => <ProgressBar aria-valuenow={10} progress={0} />})
+  const mockWarningFn = jest.fn()
+
+  beforeEach(() => {
+    jest.spyOn(global.console, 'warn').mockImplementation(mockWarningFn)
+  })
```

Comment:
Would it be possible for the `spyOn` call and mock to be only in the tests that make use of it? This would help it so that it doesn't accidentally swallow warnings from other tests.

Also wanted to add the caveat that let me know if I'm misunderstanding anything here or if we're wanting that behavior 👍